2-mandatory/3-financial-times.js

```diff
@@ -15,15 +17,26 @@ function potentialHeadlines(allArticleTitles) {
 */
 function titleWithFewestWords(allArticleTitles) {
```

Comment:
You are so close to solving the last two test cases. 
Please improve your solution and pass the last two requirements: 
✕  should return the title with the fewest words (1 ms)
✕ should return the average number of characters in a headline